Launcher/Classes/ViewControllers/TasksViewController.swift

```diff
@@ -584,14 +427,16 @@ class TasksViewController: NSViewController {
                     self.textViewPrinter.printToTextView(text)
                 }
             }
-            let commands = CommandsCore.CommandExecutor()
```

Comment:
Unrelated, let's create a ticket for this: we should remove the `runScript` from the VC.

Reply:
I've created #32 and #33